content/integrations/chef.md

```diff
@@ -5,23 +5,77 @@ kind: integration
 git_integration_title: chef
 newhlevel: true
 updated_for_agent: 5.8.5
+aliases:
+  - /guides/chef/
 description: "Track Chef client runs including metrics on completion times, analytics on resource changes, and success rates."
 ---
 
 {{< img src="integrations/chef/chefdashboard.png" alt="Chef Event" >}}
 
 ## Overview
 
-Chef is a popular configuration management tool written in Ruby and Erlang. If you manage your compute instances with Chef and want to use it to install the Datadog agent, [check out our guide for that](http://docs.datadoghq.com/guides/chef/). *This* page is about sending Chef metrics to Datadog.
+Chef is a popular configuration management tool written in Ruby and Erlang. 
 
-Connect Chef to Datadog in order to:
+Deploying Datadog with Chef is meant to be very simple, and provide you with a method of getting the value of monitoring across all of your infrastructure as simply as possible.
 
-* Get real-time reports on Chef client runs
-* Track key Chef performance metrics across all your servers
-* Quickly identify and discuss failed Chef runs with your team
+We also provide a Chef [Execution and Report Handler](https://docs.chef.io/handlers.html) that can capture `chef-client` failures as well as metrics related to the Chef run, such as timing and resources updated.
 
 ## Setup
-### Installation
+### Deploying the Agent
+
+We created a cookbook to assist with Agent deployment.
+
+Install the latest released version of the Datadog Chef cookbook from the [Community Site](https://supermarket.chef.io/cookbooks/datadog) via knife, and upload to your Chef Server:
+
+    knife cookbook site install datadog
+    knife cookbook upload datadog
+
+You may also be using another tool to manage your cookbook workflow, such as [Berkshelf](http://berkshelf.com/) or [Librarian Chef](https://github.com/applicationsonline/librarian-chef), in which case you will need to add a line like this to your Berksfile/Cheffile:
+
+    cookbook 'datadog'
```

Comment:
This is out of scope for this document. Users using Chef will know how to mange berkshelf or librarian should they wish to

Reply:
👍